engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/LazyAtomFieldTest.java

```diff
@@ -97,9 +97,9 @@ public void testInfiniteListGenerator() throws Exception {
 
     both n =
         g = Lazy.generator 1
-        // IO.println "Generator is computed"
```

Comment:
C-style line comments used to more-or-less work *under certain circumstances*, as a consequence of the operator section rules and ignored dead code. They now trigger the dead code error.